cognite/client/data_classes/geospatial.py

```diff
@@ -39,14 +40,18 @@ class FeatureTypeList(CogniteResourceList):
     _ASSERT_CLASSES = False
 
 
-class PropertyAndSearchSpec:
```

Comment:
this will break backward compatibility...
I think we still need to support the old way.

Reply:
I'll revert the change. The only way I see that can keep backward compatibility is to introduce  new method.